.travis.yml

```diff
@@ -0,0 +1,54 @@
+sudo: required
+dist: xenial
+language: scala
+jdk:
+- oraclejdk8
+before_install:
+- curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.10.1/install.sh | bash
+  && . ~/.jabba/jabba.sh
+install:
+- $JABBA_HOME/bin/jabba install $TRAVIS_JDK && export JAVA_HOME="$JABBA_HOME/jdk/$TRAVIS_JDK"
+  && export PATH="$JAVA_HOME/bin:$PATH" && java -Xmx32m -version
+before_cache:
+- find $HOME/.ivy2 -name "ivydata-*.properties" -print -delete
+- find $HOME/.sbt  -name "*.lock"               -print -delete
+cache:
+  directories:
+  - "$HOME/.ivy2/cache"
+  - "$HOME/.sbt/boot"
+  - "$HOME/.jabba/jdk"
+before_script:
+- unset _JAVA_OPTIONS
+jobs:
+  include:
+  - stage: test
+    script:
+    - sbt -J-XX:ReservedCodeCacheSize=128m test
+  - stage: int-test
+    script:
+    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+    - curl -L https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz | tar xvz --strip 1 && chmod +x oc && sudo cp oc /usr/local/bin/ && rm oc
```

Comment:
Oh, cool.

Reply:
Yeah i hadn't realised you could get it without logging in. Their docs don't mention this.